bin/tx-decoder/Cargo.toml

```diff
@@ -1,22 +1,30 @@
 [package]
 name = "tx-decoder"
-version = "0.1.0"
-edition = "2021"
+version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+description.workspace = true
+edition.workspace = true
+rust-version.workspace = true
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-desub = { path = "../../desub", features = [ "polkadot-js" ] }
+desub = { workspace = true, features = [ "polkadot-js" ] }
```

Comment:
yeah I don't like the spaces in features but could be taken care of in another PR